### PR TITLE
Feature master data

### PR DESCRIPTION
**Pull request by pierr**
## master data 

- [x] Add behaviour to set the master data config: `MasterDataProvider`, the provider put in the context the array which is named `masterDataLoaders`, you have to pass to the provider the following configuration: `configuration = [{name, service, cacheDuration}]` which will be passed in the `masterDataLoaders`
- [x] Add behaviour to set the master data config: `connectMasterData` Connect the component to the load method of the master data. You have to pass an array of names in which you are interested. These names should correspond to the configuration you provide to the `MasterDataProvider`.
- [x] Add an action to load a masterData: `loadMasterData = (name, service, cacheDuration)`
- [x] Add a reducer to set a node in the redux state under the the name `masterData`

### CODE REVIEW

**Comment by pierr** [comment]
I have my exaomples in the `git stash` but i wait for @Bernardstanislas PR to add them

File: src/behaviours/master-data.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+              if(names.indexOf(current.name) !== -1){
+                res[`loadMasterData${capitalize(current.name)}`] = () => loadMasterData(current.name, current.service, current.cacheDuration);
+              }
+              return res;
+          }, {});
+          if(Object.keys(wantedMasterDataLoaders) === 0){
+            console.warn(`connectMasterData: your keys ${Object.keys(names)} are not in the masterDataLoaders ${masterDataLoaders.reduce((res, current) => [...res, current.name]), [])}`)
+          }
+          const {_behaviours, ...otherProps} = props;
+          const behaviours = {isConnectedMasterData: true, ..._behaviours}
```
Comment (pierr) [inline src/behaviours/master-data.js]: @Bernardstanislas Shoul I add a `loadAllMasterData` props ?

Comment (Bernardstanislas) [inline src/behaviours/master-data.js]: I think that would be nice, yes

Comment (Bernardstanislas) [inline src/behaviours/master-data.js]: I don't get your reduce, wouldn't it be : `loadAllMasterData = () => Object.keys(wantedMasterDataLoaders).reduce((res, current) => wantedMasterDataLoaders[current]());`

**Pull request merged**